page.php

```diff
@@ -21,18 +21,35 @@
  * @since    Timber 0.1
  */
 
+$copyrightText = get_option( 'copyright' );
+
+// Get Menu items using their names.
+$footerSocialMenuName     = 'Footer Social';
+$footerPrimaryMenuName    = 'Footer Primary';
+$footerSecondaryMenuName  = 'Footer Secondary';
+
+$footerSocialMenuItems    = wp_get_nav_menu_items( $footerSocialMenuName );
+$footerPrimaryMenuItems   = wp_get_nav_menu_items( $footerPrimaryMenuName );
+$footerSecondaryMenuItems = wp_get_nav_menu_items( $footerSecondaryMenuName );
+
```

Comment:
These vars only used once, in the lines below. It's more concise to use them directly where they needed. So remove lines 24-34.. and add the values to lines 47-49

Reply:
Done